quartz_solar_forecast/data.py

```diff
@@ -111,20 +114,45 @@ def format_nwp_data(df: pd.DataFrame, nwp_source:str, site: PVSite):
     )
     return data_xr
 
+def get_enphase_data(enphase_user_id: str, enphase_api_key: str) -> float:
+    """
+    Get live PV generation data from Enphase API
 
-def make_pv_data(site: PVSite, ts: pd.Timestamp) -> xr.Dataset:
+    :param enphase_user_id: User ID for Enphase API
+    :param enphase_api_key: API Key for Enphase API
+    :return: Live PV generation in Watt-hours, assumes to be a floating-point number
     """
-    Make fake PV data for the site
+    url = f'https://api.enphaseenergy.com/api/v2/systems/{enphase_user_id}/summary'
+    headers = {'Authorization': f'Bearer {enphase_api_key}'}
+
+    response = requests.get(url, headers=headers)
+    data = response.json()
+
+    # Extracting live generation data assuming it's in Watt-hours
+    live_generation_wh = data['current_power']['power']
+
+    return live_generation_wh
+
+def make_pv_data(site: PVSite, ts: pd.Timestamp, use_enphase_data: bool = False) -> xr.Dataset:
```

Comment:
need to propogate this through to the top. Perhaps the site object could have an option field with is `inverter` and then if its not None, then we use the `use_enphase_data` flag

Reply:
> need to propogate this through to the top. Perhaps the site object could have an option field with is `inverter` and then if its not None, then we use the `use_enphase_data` flag

Updated the site object to accomodate this in latest push #66